link/link.go

```diff
@@ -56,6 +58,19 @@ func NewLinkFromFD(fd int) (Link, error) {
 	return wrapRawLink(&RawLink{fd: sysFD})
 }
 
+// NewLinkFromID returns the link associated with the given id.
+//
+// Returns ErrNotExist if there is no link with the given id.
+func NewLinkFromID(id ID) (Link, error) {
```

Comment:
Nit: this reads as `link.NewLinkFromID`, maybe just `link.NewFromID` is nicer?

Reply:
I think this was named after `link.NewLinkFromFD` for consistency; should we change both?

Reply:
Sounds good, I changed both.

Reply:
Made it so that we keep the old function around but `Deprecate` it. No need to break users in this simple case.